libs/category/driver/shopify/src/queries/gql/get-category-by-url.ts

```diff
@@ -0,0 +1,51 @@
+import { gql } from 'apollo-angular';
+
+import { ShopifyCategoryResponse } from '../response.types';
+import { ShopifyCategoryUrlVariables } from '../variables.types';
+
+/**
+ * GraphQL query object for getting a category (Shopify Collection) by url (handle).
+ */
+export const getCategoryByUrl = gql<ShopifyCategoryResponse, ShopifyCategoryUrlVariables>`
+  query GetCategoryByUrl($handle: String!, $reverse: Boolean, $sortKey: ProductCollectionSortKeys, $filters: [ProductFilter!]!) {
+    collection(handle: $handle) {
+      handle
+      id
+      title
+      description
+      onlineStoreUrl
+      image { 
+        altText 
+        id 
+        url 
+      } 
+      products(first: 10, reverse: $reverse, sortKey: $sortKey, filters: $filters) {
+        nodes { 
+          handle
+          onlineStoreUrl
+          availableForSale
+          priceRange { 
+            maxVariantPrice { 
+              amount 
+              currencyCode 
+            } 
+            minVariantPrice { 
+              amount 
+              currencyCode 
+            } 
+          } 
+          id
+          title
+          description
+          images(first: 1) { 
+            nodes { 
+              id 
+              url 
+              altText 
+            } 
+          } 
```

Comment:
This type should be broken down into a few fragments. I'd say you should make `Collection`, `Image`, `Product`, and `Price` fragments.